prepare_data/README.md

```diff
@@ -13,6 +13,8 @@ In its current state, this pipeline uses [SCT development version](https://githu
 See [README](../README.md)
 ~~~
 source PATH_TO_YOUR_VENV/venv-ivadomed/bin/activate
+# install prepdata
+cd .. && python setup.py install && cd -
```

Comment:
It seems that [prepdata](https://github.com/neuropoly/ivado-medical-imaging/tree/master/prepare_data/prepdata) is not installed with ivadomed, which leads to the failure of `./run_process.sh parameters.sh final_qc.sh`.

Reply:
ah yes you're right... it's not the most intuitive way of organizing a software structure (i'm the one to blame), and the only dependent python software is actually `prepdata remove-slice` (called by `final_qc.sh`). So i guess until we find a better structure/organization (feel free to suggest), we should indeed keep that install documentation.